CONTRIBUTING.md

```diff
@@ -96,6 +96,31 @@ Some components may require specific setup steps. Please refer to their respecti
 - [Node.js Binding](bindings/nodejs/CONTRIBUTING.md)
 - [Python Binding](bindings/python/CONTRIBUTING.md)
 
+## Build from source
+
+OpenDAL uses [Cargo](https://doc.rust-lang.org/cargo/) as its build system. 
+To build OpenDAL, run the following command under OpenDAL's root directory:
+
+Only build the core component:
+
+```shell
+cargo build
+```
+
+With full features enabled:
+
+```shell
+cargo build --all-features
+```
```

Comment:
It's almost impossible to pass without a special setup, especially for HDFS. I don't believe the Contributing Guide is an appropriate place for this.

Reply:
Make sense.